addons/survey/static/src/js/survey_result.js

```diff
@@ -55,18 +56,32 @@ publicWidget.registry.SurveyResultPagination = publicWidget.Widget.extend({
 
         var $target = $(ev.currentTarget);
         $target.closest('li').addClass('active');
-        this.$questionsEl.find('tbody tr').addClass('d-none');
+        this.$questionsEl.find("tbody tr").addClass("d-none");
 
         var num = $target.text();
-        var min = (this.limit * (num-1))-1;
-        if (min === -1){
-            this.$questionsEl.find('tbody tr:lt('+ this.limit * num +')')
-                .removeClass('d-none');
+        var min = this.limit * (num - 1) - 1;
+        if (min === -1) {
+            this.$questionsEl
+                .find("tbody tr:lt(" + this.limit * num + ")")
+                .removeClass("d-none");
         } else {
-            this.$questionsEl.find('tbody tr:lt('+ this.limit * num +'):gt(' + min + ')')
-                .removeClass('d-none');
+            this.$questionsEl
+                .find("tbody tr:lt(" + this.limit * num + "):gt(" + min + ")")
+                .removeClass("d-none");
         }
+    },
 
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onShowAllAnswers: function (ev) {
+        const btnEl = ev.currentTarget;
+        const pager = btnEl.previousElementSibling;
+        btnEl.classList.add("d-none");
+        this.$questionsEl.find("tbody tr").removeClass("d-none");
+        pager.classList.add("d-none");
+        this.$questionsEl.parent().attr("style", "auto");
```

Comment:
"style" "auto" seems a bit strange, not sure what this does.

If this is to ensure the height is set to auto, I think you can instead add the "h-auto" bootstrap class.

Reply:
Sorry, this is a typo. Should be `height: auto`. Ok for `h-auto` class